simhash/__init__.py

```diff
@@ -16,6 +16,10 @@
     range = xrange
 
 
+# prepare module logger
+log = logging.getLogger("simhash")
```

Comment:
what's the benefit of putting `log` here rather than putting it as a parameter of `Simhash.__init__` function?

Reply:
Like this it is global and could be used also in other functions, such as `_hashfunc`, in the module; however, if you want to put it inside the `Simhash` class it would also be okay.

Reply:
good point. think we can also make `log` a parameter of `_hashfunc`?
I would prefer putting `log` as an optional parameter of `Simhash.__init__`, because it's more explicit.